Gems/Atom/RHI/DX12/Code/Source/RHI/StreamingImagePool.h

```diff
@@ -65,10 +68,26 @@ namespace AZ
             // Packed mips occupy a dedicated set of tiles.
             void AllocatePackedImageTiles(Image& image);
 
-            RHI::Ptr<ID3D12Heap> m_heap;
+            // Get the data reference of device heap memory usage 
+            RHI::HeapMemoryUsage& GetDeviceHeapMemoryUsage();
+
+            // whether to enable tiled resource
```

Comment:
Whether tile resource support is available.

Reply:
it actually mean enable or disable. the availability of tile resource is provided by device.GetFeatures().m_tiledResource.